.github/workflows/self_hosted_e2e.yaml

```diff
@@ -23,23 +23,43 @@ jobs:
       ARCHIVE_OUTDIR: dist/archives
     strategy:
       matrix:
-        os: [ubuntu-latest]
-        target_arch: [amd64, arm64]
+        os: [ubuntu-latest, macos-latest]
+        target_arch: [amd64]
         include:
           - os: ubuntu-latest
             target_os: linux
-          #- os: windows-latest
-          #  target_os: windows
-          #- os: macOS-latest
-          #  target_os: darwin
+          - os: macOS-latest
+            target_os: darwin
+            #- os: windows-latest
+            #target_os: windows
     steps:
       - name: Set up Go ${{ env.GOVER }}
         uses: actions/setup-go@v2
         with:
           go-version: ${{ env.GOVER }}
       - name: Check out code into the Go module directory
         uses: actions/checkout@v2
-      - name: Setup Docker
+      - name: Setup Docker - Linux
```

Comment:
is this step needed at all since arm64 is disabled? 
Just comment it out for now? Can be added back once arm64 tests are setup?

Reply:
You are right, we don't need that anymore.